src/__integrationtests__/Ctypes.spec.ts

```diff
@@ -16,7 +17,7 @@ import { wannabeFaucet } from './utils'
 
 let blockchain: IBlockchainApi | undefined
 beforeAll(async () => {
-  blockchain = await getCached(DEFAULT_WS_ADDRESS)
+  blockchain = await getCached(getNodeAddress())
```

Comment:
same here, I'd expect this to be done implicitly - and error if the config wasn't set